src/components/Capture/Face.js

```diff
@@ -59,10 +63,24 @@ class Face extends Component {
     callback()
   }
 
+  handleError = () => this.props.actions.deleteCapture()
+
+  handleFileSelected = (file) => {
```

Comment:
This is duplicated, same function is in Document.js - maybe extract it to separate utils file?